tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp

```diff
@@ -273,3 +308,13 @@ SPECTRE_TEST_CASE("Unit.ApparentHorizons.StrahlkorperGr.RicciScalar",
       },
       [](const size_t size) noexcept { return DataVector(size, 0.5); });
 }
+
+SPECTRE_TEST_CASE("Unit.ApparentHorizons.StrahlkorperGr.AreaElement",
+                  "[ApparentHorizons][Unit]") {
+  test_area_element(
```

Comment:
please add a test for a spinning non-unit mass Kerr hole;  I believe the area is 4*PI*(r_+^2 + a^2) (Eq 5.54 of Poisson's Relativist's Toolkit)